src/client/app/+home/home.component.html

```diff
@@ -1,12 +1,7 @@
-<p>
-  Howdy! Here's a list of awesome computer scientists. Do you know any others? Add to the list yourself.
-</p>
-
-<form (submit)="addName()">
-  <input [(ngModel)]="newName" name="newName" placeholder="Awesome Computer Scientist">
-  <button type="submit">Add</button>
-</form>
-
 <ul>
-  <li *ngFor="let name of names">{{name}}</li>
-</ul>
+  <li *ngFor="let card of cards">
+    <a [routerLink]="['/graph', card.name]" [routerLinkActive]="['router-link-active']" [routerLinkActiveOptions]="{exact:true}">
```

Comment:
(nit)

Personally, I like to align each of the HTML elements next to each other so that they create a nice vertical line with each other.